shared/src/popup.js

```diff
@@ -306,10 +306,26 @@ async function setup() {
     window.close();
   }
 
-  requestPermissionsButton.addEventListener(
-    'click',
-    handleRequestPermissionsButtonClick,
-  );
+  const platformInfo = await browser.runtime.getPlatformInfo();
+  const browserInfo =
+    typeof browser.runtime.getBrowserInfo === 'function' &&
+    (await browser.runtime.getBrowserInfo());
+
+  // Note, _hoping_ by 119 this works, but there's no guarantee.
```

Comment:
What happens if by 119 it still doesn't work, is that clicking the button won't do anything (like what happens for 0.3.6 right now).